app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split()
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names